bgpd/bgp_evpn.c

```diff
@@ -5447,11 +5447,16 @@ int bgp_evpn_local_l3vni_add(vni_t l3vni, vrf_id_t vrf_id, struct ethaddr *rmac,
 	if (filter)
 		SET_FLAG(bgp_vrf->vrf_flags, BGP_VRF_L3VNI_PREFIX_ROUTES_ONLY);
 
-	/* auto derive RD/RT */
+	/* Map auto derive or configured RTs */
 	if (!CHECK_FLAG(bgp_vrf->vrf_flags, BGP_VRF_IMPORT_RT_CFGD))
 		evpn_auto_rt_import_add_for_vrf(bgp_vrf);
+	else
```

Comment:
Similar check and code not required for EXPORT RT ? I don't see similar modification for EXPORT RT below. 

Otherwise, this code changes looks good.

Reply:
for EXPORT RT it is not applicable.